app/policies/booking_policy.rb

```diff
@@ -0,0 +1,35 @@
+class BookingPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
+    record.user == user
```

Comment:
should probably be true since you're calling scope on index anyways 🤷🏻‍♂️

Reply:
to be more specific / clear.
I think it makes sense to get a boolean for a single item.
"can I show this booking ?" "Yes, show if the booking belongs to that user"
But we cannot have something like
"can I show all bookings ?" "Yes, show all the bookings if all the bookings belong to that user"
This is why if you use 
```
def index?
  true
end
```
you're saying "can I show all the bookings ?" => "Yes"
and then you're calling scope which basically says: "What are all the bookings?" "It's all the bookings where the user is the current user"